### PR TITLE
Add help text to sections on GPS tab

### PR DESCRIPTION
**Pull request by HThuren**
This PR add help text to sections on GPS tab

### CODE REVIEW

File: locales/en/messages.json [hunk 1]
```diff
@@ -2689,9 +2693,17 @@
...
     },
+    "gpsHeadHelp": {
+        "message": "Show various information releated to GPS. Heading are only shown when Magnetometer are in use",
+        "description": "Help text for gpsHead"
+    },
     "gpsMapHead": {
         "message": "Current GPS location"
     },
+    "gpsMapHeadHelp": {
+        "message": "Show GPS location on map. Use mouse to zoom in and out.<br>Location icon show North direction when Magnetometer are in use",
```
Comment (haslinghuis) [inline locales/en/messages.json:2704]:
```suggestion
        "message": "Show GPS location on the map. Use mouse scroll button to zoom in and out.<br>The location icon shows the north direction when the magnetometer is in use",
```

Comment (HThuren) [reply locales/en/messages.json:2704]: Better english :-)

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by haslinghuis** [comment]
Thanks @HThuren. Can we divert our attention to solving existing feature requests aiming to bring down number of issues :)

**Comment by HThuren** [comment]
> Thanks @HThuren. Can we divert our attention to solving existing feature requests aiming to bring down number of issues :)

Well, havn't thought on that. Why not :-)

**Pull request merged**